projects/step-frontend/src/lib/modules/timeseries/chart/time-series-chart.component.scss

```diff
@@ -4,3 +4,60 @@
   max-height: 100%;
   padding-top: 24px;
 }
+
```

Comment:
Style encapsulation turned off for this components.
This style may intersect with some other styles in applications.
If `ViewEncapsulation.None` is required for current case, it make sense add parent selector for each rule, to prevent intersection. The simplest way would be just to wrap styles as following:
```
step-timeserises-chart {
    
   .chart-wrapper {
             ....
   }
  
  .ranger {
    ...
  }

  .ts-tooltip {
    ...
  }

 ...
}
```

Reply:
will fix this!